src/Microsoft.DotNet.BuildTools.TestSuite/runtime.json

```diff
@@ -0,0 +1,95 @@
+{
+    "supports": {
```

Comment:
Would be nice if we could generate this file from some common mapping.  As it stands we have the TFM:RID mappings duplicated here and [here](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L840).  We have the RID list for netcoreapp here https://github.com/dotnet/corefx/blob/12fee7b6e85fce4100856907246a0e1056e64e0c/pkg/Microsoft.NETCore.Platforms/runtime.json.

Not needed for this PR.